packages/block-library/src/navigation-menu/index.php

```diff
@@ -75,32 +75,34 @@ function render_block_navigation_menu( $attributes, $content, $block ) {
 function build_navigation_menu_html( $block, $colors ) {
 	$html = '';
 	foreach ( (array) $block['innerBlocks'] as $key => $menu_item ) {
+		$title            = isset( $menu_item['attrs']['title'] ) ? esc_attr( $menu_item['attrs']['title'] ) : '';
+		$label            = isset( $menu_item['attrs']['label'] ) ? esc_attr( $menu_item['attrs']['label'] ) : '';
+		$url              = isset( $menu_item['attrs']['url'] ) ? esc_attr( $menu_item['attrs']['url'] ) : '';
+		$opens_in_new_tab = isset( $menu_item['attrs']['opensInNewTab'] ) && true === $menu_item['attrs']['opensInNewTab'];
 
-		$html .= '<li class="wp-block-navigation-menu-item">' .
-			'<a
-				class="wp-block-navigation-menu-item__link ' . $colors['text_css_classes'] . '"
-				' . $colors['text_inline_styles'];
-
-		// Start appending HTML attributes to anchor tag.
+		// Creates the markup for the item content element.
 		if ( isset( $menu_item['attrs']['url'] ) ) {
-			$html .= ' href="' . $menu_item['attrs']['url'] . '"';
-		}
-		if ( isset( $menu_item['attrs']['title'] ) ) {
-			$html .= ' title="' . $menu_item['attrs']['title'] . '"';
+			$item_content_markup =
+				'<a
+					class="wp-block-navigation-menu-item__link ' . $colors['text_css_classes'] . '"
+					' . $colors['text_inline_styles'] .
+					' href="' . $url . '"' .
+					( $title ? ' title="' . $title . '"' : '' ) .
+					( $opens_in_new_tab ? ' target="_blank"' : '' ) .
+				'>' .
+					( $label ? $label : '' ) .
+				'</a>';
+		} else {
+			$item_content_markup =
+				'<span
+					class="wp-block-navigation-menu-item__text ' . $colors['text_css_classes'] . '"
+					' . $colors['text_inline_styles'] .
+				'>' .
+					( isset( $menu_item['attrs']['label'] ) ? $label : '' ) .
```

Comment:
This is the same `$label` as above right?

Reply:
yes :-( Going to update